### ISSUE TITLE
bg_cnt=0 encouters division by zero

### ISSUE THREAD

**Issue by zchrissirhcz** [longcw/faster_rcnn_pytorch#53]
In your `faster_rcnn.py`, the cross entropy loss has a weight, which is calculate as:
```
ce_weigts[0] = float(fg_cnt) / bg_cnt
```

When using my own training data (which, can be correctly trained on original py-faster-rcnn), will encounter problems. First, when calculating `self.tf`(in the same file), will access invalid array index; Second, the cross entropy weight will be zero.

Hopy this is useful for people that would like to use their own dataset.

**Comment by Jongchan** [comment]
Same here. Did you find any other way around?

**Comment by Jongchan** [comment]
Are you using a customized imdb dataset loader (as in faster_rcnn/datasets/)
In my case, I have created a new imdb sub class for my dataset.
